components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -59,6 +60,19 @@ class GeckoEngine(
         executor.speculativeConnect(url)
     }
 
+    /**
+     * See [Engine.installWebExtension].
+     */
+    override fun installWebExtension(ext: WebExtension) {
+        fakeRuntime.registerWebExtension(org.mozilla.geckoview.WebExtension(ext.id, ext.url))
```

Comment:
If you want to avoid the name confusion here then you can import the type with a different name:

```
import org.mozilla.geckoview.WebExtension as GeckoWebExtension
```

https://kotlinlang.org/docs/reference/packages.html#imports

Reply:
OK, cleaned this up!